src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketScmFormValidationDelegate.java

```diff
@@ -86,8 +97,7 @@ public FormValidation doCheckProjectName(@Nullable Item context, String serverId
                                 .getClient(
                                         serverConf.getBaseUrl(),
                                         jenkinsToBitbucketCredentials.toBitbucketCredentials(
-                                                providedCredentials.orElse(null),
-                                                serverConf.getGlobalCredentialsProvider("Check Project Name")));
+                                                providedCredentials.orElse(null)));
```

Comment:
You know that `providedCredentials` is present, so its safe to call `.get`

Reply:
- [ ] call `.get()` instead